fontbe/src/glyphs.rs

```diff
@@ -686,6 +691,34 @@ impl CheckedGlyph {
             CheckedGlyph::Contour { .. } => true,
         }
     }
+
+    /// Flip the glyph contours' direction, or do nothing if the glyph is a composite.
+    ///
+    /// The source contours are normally drawn with cubic curves thus are expected to be
+    /// in counter-clockwise winding direction as recommended for PostScript outlines.
+    /// When converting to TrueType quadratic splines, we reverse them so that they
+    /// follow the clockwise direction as recommeded for TrueType outlines.
+    fn reverse_contour_direction(&mut self) {
+        let CheckedGlyph::Contour {
+            name,
+            paths: contours,
+        } = self
+        else {
+            return; // nop for composite
+        };
+
+        trace!("Reverse '{name}' contour direction");
+
+        let reversed = contours
+            .iter()
+            .map(|(loc, contour)| (loc.clone(), contour.reverse_subpaths()))
+            .collect();
+
+        *self = CheckedGlyph::Contour {
+            name: name.clone(),
+            paths: reversed,
+        }
```

Comment:
```suggestion
        match self {
            CheckedGlyph::Contour { name, paths } => {
                trace!("Reverse '{name}' contour direction");
                for (_, contour) in paths {
                    *contour = contour.reverse_subpaths();
                }
            }
            _ => {} // nop for composite
        }
```
This might be a bit more clear? If you match on an enum value with `&mut` binding, the internal fields of the variants are also bound `&mut` in the pattern so you can modify them directly.